src/middlewares/build-include.ts

```diff
@@ -0,0 +1,84 @@
+import { Response, NextFunction } from "express";
+import { SequelizeQueryParserRequestInterface, IncludeObject } from "../core/interfaces";
+import {
+  INVALID_INCLUDE,
+  MODEL_NOT_CONFIGURED_ERROR,
+  SEQUELIZE_QUERY_PARSER_DATA_NOT_FOUND_ERROR,
+} from "../core/constants";
+import { parseStringWithParams } from "../utils";
+import { hasIn } from "lodash";
+
+/**
+ * Middleware to build the include array for Sequelize queries based on request parameters.
+ * @param req - Express request object with SequelizeQueryParserRequestInterface
+ * @param res - Express response object
+ * @param next - Express next function
+ * @throws {Error} SEQUELIZE_QUERY_PARSER_DATA_NOT_FOUND_ERROR if necessary Sequelize query parser data is missing
+ * @throws {Error} MODEL_NOT_CONFIGURED_ERROR if the model is not configured in the request
+ */
+export function buildInclude(
+  req: SequelizeQueryParserRequestInterface,
+  res: Response,
+  next: NextFunction
+) {
+  // Check if necessary Sequelize query parser data exists
+  if (!req.sequelizeQueryParser)
+    throw new Error(SEQUELIZE_QUERY_PARSER_DATA_NOT_FOUND_ERROR);
+  if (!req.sequelizeQueryParser.model)
+    throw new Error(MODEL_NOT_CONFIGURED_ERROR);
+
+  const model = req.sequelizeQueryParser.model;
+  let include;
+  if (!req.query.include) {
+    include = [];
```

Comment:
Include is an optional property in query parser interface. Just call next here instead of initialize an empty array